docs/content/build.md

```diff
@@ -58,3 +58,14 @@ This starts the build process for the worker and deployment type (cloud or local
 > **It’s finished building when:** there is a message stating `Completed build for Local/Cloud target`.
 
 You can now find the built-out workers in `build/assembly/worker` in the root of the SpatialOS GDK for Unity project.
+
+#### TIP: Speed up development iteration with worker quick-run
+ When you are developing with the GDK, you don't need to build out workers all the time, you can use quick-run to run multiple workers in your Unity Editor using Ctrl+L (Windows) or Cmd+L (Mac).
+
+When you are developing with the GDK, you use a local deployment rather than a cloud deployment. In a local deployment, you can either build your workers to run locally or use quick-run. With quick-run you can run multiple workers in your Unity Editor, so you don't have to keep building out workers during development iteration.  
```

Comment:
```suggestion
During development , you use a local deployment rather than a cloud deployment. In a local deployment, you can either build your workers to run locally or use quick-run. With quick-run you can run multiple workers in your Unity Editor, so you don't have to keep building out workers during development iteration.  
```